plugin/db/util/driverutil_test.go

```diff
@@ -278,6 +278,21 @@ func TestExtractSensitiveField(t *testing.T) {
 		schemaInfo *db.SensitiveSchemaInfo
 		fieldList  []db.SensitiveField
 	}{
+		{
+			// Test for associated sub-query.
+			statement:  `select a, (select max(b) > y.a from t as x) from t as y`,
+			schemaInfo: defaultDatabaseSchema,
+			fieldList: []db.SensitiveField{
+				{
+					Name:      "a",
+					Sensitive: true,
+				},
+				{
+					Name:      "(select max(b) > y.a from t as x)",
```

Comment:
Just for curiosity, why will this column be sensitive? Is it because it could potentially leak information about column a`?

Reply:
Thx